cypress/component/DataSubmission/consent_group.spec.js

```diff
@@ -0,0 +1,59 @@
+import ConsentGroupForm from '../../../src/components/data_submitter/ConsentGroupForm';
+import { cloneDeep } from 'lodash/fp';
+import { mount } from 'cypress/react';
+
+const props = {
+  idx: 0,
+  parentConsentGroup: {},
+  saveConsentGroup: () => {},
+  deleteConsentGroup: () => {},
+}
+
+let propCopy = {}; 
+
+beforeEach(() => {
+    propCopy = cloneDeep(props);
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/DataBiosphere/duos-ui/pullRequest?prid=9969451)
```suggestion
  propCopy = cloneDeep(props);
```